components/match2/wikis/apexlegends/game_summary.lua

```diff
@@ -0,0 +1,261 @@
+
+	local gameDetails = page:tag('div')
+			:addClass('panel-content__container')
+			:attr('id', 'panelContent1')
+			:attr('role', 'tabpanel')
+
+	local informationList = gameDetails:tag('ul'):addClass('panel-content__game-schedule')
+	informationList:tag('li')
+			:tag('div')
+					:addClass('panel-content__game-schedule__container')
+					:node(SummaryHelper.countdownIcon(game, 'panel-content__game-schedule__icon'))
+					:node(SummaryHelper.gameCountdown(game))
+	if game.map then
+		informationList:tag('li')
+				:tag('i')
+						:addClass('far fa-map')
+						:addClass('panel-content__game-schedule__icon')
+						:done()
+				:tag('span'):wikitext(Page.makeInternalLink(game.map))
+	end
+
+	page:node(SummaryHelper.createPointsDistributionTable(scoreData))
+
+	return page:node(CustomGameSummary._createGameStandings(game))
+end
+
+---@param game table
+---@return Html
+function CustomGameSummary._createGameStandings(game)
+	local wrapper = mw.html.create('div')
+			:addClass('panel-table')
+			:attr('data-js-battle-royale', 'table')
+	local header = wrapper:tag('div')
+			:addClass('panel-table__row')
+			:addClass('row--header')
+			:attr('data-js-battle-royale', 'header-row')
+
+	Array.forEach(GAME_STANDINGS_COLUMNS, function(column)
+		local cell = header:tag('div')
+			:addClass('panel-table__cell')
+			:addClass(column.class)
+			local groupedCell = cell:tag('div'):addClass('panel-table__cell-grouped')
+				:tag('i')
+						:addClass('panel-table__cell-icon')
+						:addClass(column.iconClass)
+						:done()
+				:tag('span')
+						:wikitext(column.header.value)
+						:done()
+			if (column.sortable and column.sortType) then
+				cell:attr('data-sort-type', column.sortType)
+				groupedCell:tag('div')
+					:addClass('panel-table__sort')
+					:tag('i')
+						:addClass('far fa-arrows-alt-v')
+						:attr('data-js-battle-royale', 'sort-icon')
+			end
+	end)
+
+	Array.forEach(game.extradata.opponents, function (opponent, index)
+		local row = wrapper:tag('div'):addClass('panel-table__row'):attr('data-js-battle-royale', 'row')
+		Array.forEach(GAME_STANDINGS_COLUMNS, function(column)
+			local cell = row:tag('div')
+					:addClass('panel-table__cell')
+					:addClass(column.class)
+					:node(column.row.value(opponent, index))
+			if (column.sortType) then
+				cell:attr('data-sort-val', column.sortVal.value(opponent, index)):attr('data-sort-type', column.sortType)
+			end
+		end)
+	end)
+	return wrapper
+end
+
+function CustomGameSummary._opponents(match)
+	-- Add match opponent data to game opponent and the other way around
+	Array.forEach(match.games, function (game)
+		game.extradata.opponents = Array.map(game.extradata.opponents, function (opponent, opponentIdx)
+			local opp = Table.merge(match.opponents[opponentIdx], opponent)
+			-- These values are only allowed to come from Game and not Match
+			opp.placement = opponent.placement
+			opp.score = opponent.score
+			opp.status = opponent.status
+			return opp
```

Comment:
preferably name them matchOpponent and gameOpponent instead of opp and opponent
seems cleaner to read